packages/plugin-google-analytics/test/cases/default/default.spec.js

```diff
@@ -1,116 +1,116 @@
 /*
  * Use Case
  * Run Greenwood with Google Analytics composite plugin with default options.
- * 
+ *
  * Uaer Result
  * Should generate a bare bones Greenwood build with Google Analytics tracking snippet injected into index.html.
- * 
+ *
  * User Command
  * greenwood build
- * 
+ *
  * User Config
  * const googleAnalyticsPlugin = require('@greenwod/plugin-google-analytics');
- * 
+ *
  * {
  *   plugins: [{
  *     ...googleAnalyticsPlugin({
  *       analyticsId: 'UA-123456-1'
  *     })
  *  }]
- * 
+ *
  * }
- * 
+ *
  * User Workspace
  * Greenwood default (src/)
  */
-const expect = require('chai').expect;
-const { JSDOM } = require('jsdom');
-const path = require('path');
+// const expect = require('chai').expect;
+// const { JSDOM } = require('jsdom');
+// const path = require('path');
 const runSmokeTest = require('../../../../../test/smoke-test');
 const TestBed = require('../../../../../test/test-bed');
 
 describe('Build Greenwood With: ', async function() {
   const LABEL = 'Google Analytics Plugin with default options and Default Workspace';
-  const mockAnalyticsId = 'UA-123456-1';
+  // const mockAnalyticsId = 'UA-123456-1';
 
   let setup;
 
   before(async function() {
     setup = new TestBed();
-    this.context = setup.setupTestBed(__dirname);
+    this.context = await setup.setupTestBed(__dirname);
   });
-  
+
   describe(LABEL, function() {
-    before(async function() {     
+    before(async function() {
       await setup.runGreenwoodCommand('build');
     });
-    
+
     runSmokeTest(['public', 'index', 'not-found', 'hello'], LABEL);
 
-    describe('Initialization script at the end of the <body> tag', function() {
-      let inlineScript;
+    // describe('Initialization script at the end of the <body> tag', function() {
```

Comment:
instead of commenting out like this, in a test, to help with readability and review from a PR perspective, I would request using `xdescribe` (or `xit`) for disabling test code.  Less line changes / diffs and makes it clearer and cleaner that a test or suite is being ignored.

Reply:
Ah that was it, I forgot the syntax at that moment.

Reply:
Oh right, the problem with this is that it will fail with exit code 1 still with each of the xdescribe tests listed as "pending".

Reply:
`describe.skip()` works though